techtv2ovs/utils.py

```diff
@@ -155,6 +159,23 @@ def run(self):
                 self.etl_graph()
             )
 
+    def get_stream_source(self, collection_id):
+        """
+        Return the appropriate stream_source for transcoded videos
+        Args:
+            collection_id (int): The collection id
+
+        Returns:
+            StreamSource: where the video should be streamed from - Youtube, Cloudfront, or either one (None)
+        """
+        if self.protected == 1:
```

Comment:
Is this because youtube videos can't be protected? If so why wouldn't this return `CLOUDFRONT`?

Reply:
Correct.  Once `ENABLE_VIDEO_PERMISSIONS` is set to `True`, people will be able to manually  override collection-level permissions for individual videos and make them public, and I think in that case we want it to stream from Youtube if available, Cloudfront otherwise (`stream_source` = `None`). @pdpinch let me know if I've got that wrong.